### DISCUSSION TITLE
Selenide into Selenium Project

### DISCUSSION THREAD

**Discussion by NagarjunaSK-Git** [selenide/selenide#2740] [Ideas]
Selenide is wonderfully concise and why this can't be out of the box from Selenium project itself?

**Comment by asolntsev** [comment]
Thank you, I also wish that! :) 

Because these two projects have different goals:
1. Selenium is NOT a testing library. It's a tool for driver a browser (so called webdriver).
2. Selenide IS a library for testing. 

In fact, if you create a test automation project in IDEA, Selenide is selected by default:

File -> New -> Project -> "Selenium" -> 

<img width="802" alt="image" src="https://github.com/selenide/selenide/assets/279773/ad9ec57d-1b71-473f-834f-162e85d5b7b9">
